content/applications/finance/payment_acquirers/stripe.rst

```diff
@@ -61,9 +54,10 @@ Odoo.sh or On-premise
 .. _stripe/api_keys:
 
 Publishable and Secret keys
```

Comment:
Following the comment above, this heading and the next one could be an exact copy of the instructions that we gave in "Link your Stripe Account with Odoo"
```suggestion
Fill in your credentials
```
and
`Generate a webhook`